.travis/run_travis_job.sh

```diff
@@ -34,7 +34,7 @@ echo 'Performing checks over the rust code'
 cargo fmt --all -- --check
 
 # Run clippy static analysis.
-cargo clippy --all --tests --all-features -- -D warnings
+cargo clippy --all --tests --all-features -- -D warnings -A deprecated
```

Comment:
Can this be allowed at the test/function level? I think we don't want to allow app deprecated methods.